doc/classes/RichTextLabel.xml

```diff
@@ -525,7 +546,7 @@
 		</constant>
 		<constant name="ITEM_RAINBOW" value="20" enum="ItemType">
 		</constant>
-		<constant name="ITEM_CUSTOMFX" value="22" enum="ItemType">
+		<constant name="ITEM_CUSTOMFX" value="23" enum="ItemType">
```

Comment:
Seems like the order of bindings doesn't match the header, and some values are not exposed.

Reply:
Fixed.